services/frontend-service/src/ui/components/ApplicationLockDisplay/ApplicationLockDisplay.tsx

```diff
@@ -0,0 +1,67 @@
+/*This file is part of kuberpult.
+
```

Comment:
does it make sense to combine the two `Display` components? 

all locks are similar, and they should be displayed the same way?

Reply:
Yeah, it makes sense of making into one component. Already did that!